### ISSUE TITLE
error with WPML

### ISSUE THREAD

**Issue by zabetta** [Automattic/WP-Job-Manager#1176]
website : http://zhermackhr.nomesia.com, multilanguage with wpml

if I try to update wpml main plugin to last version 4.8.1 i can't see active job list
here http://zhermackhr.nomesia.com/en/jobs/  and here zhermackhr.nomesia.com/en/application/

The system seems to run with WPML 3.7.1 , currently installed

**Comment by rachelsquirrel** [comment]
Also reported in 689164-zen

**Comment by xxhenryxx** [comment]
What meaning that?

**Comment by htdat** [comment]
## TL;DR

This is a WPML issue in filtering the [home_url](http://codex.wordpress.org/Function_Reference/home_url) function. There is an example: 

```
$url = home_url( 'example', 'relative' );
echo esc_url( $url );
``` 

* Correct result without WPML: `/example`. 
* Wrong result with WPML: `example` (mising slash `/` is important here). 

**We suggest contacting with WPML to resolve this issue**. 

## This is from my debug. 

### My stack: 

* WPML 3.8.1 - latest
* WP Job Manager 1.28.0 - latest
* WordPress 4.8.1 

### What I see: 

* My English job dashboard http://my-domain.me/jobs/ looks fine. In the source code, there is this code: 

```
var job_manager_ajax_filters = {"ajax_url":"\/jm-ajax\/%%endpoint%%\/","is_rtl":"0","i18n_load_prev_listings":"Load previous listings","lang":"en"};
```

* My Spanish http://my-domain.me/es/jobs/ looks broken - see the screenshot below. 

```
var job_manager_ajax_filters = {"ajax_url":"http:\/\/jm-ajax\/%%endpoint%%\/","is_rtl":"0","i18n_load_prev_listings":"Load previous listings","lang":"es"};
```

* The broken part is `http:\/\/` when using the 2nd language. 

* Further debugging results into the [WP_Job_Manager_Ajax::get_endpoint](https://github.com/Automattic/WP-Job-Manager/blob/0e1956f4858f15d0ce900fa41dd0cb66ef579798/includes/class-wp-job-manager-ajax.php#L71
) method. 

* Even further, it results in the WPML issue with `home_url()` above. 

---- 

Screenshot: 
![2017-09-20_10-25-48](https://user-images.githubusercontent.com/10045087/30625575-3cdc167e-9dee-11e7-81fb-943f7cfe2828.png)

**Comment by xxhenryxx** [comment]
Sorry, but i dont know exactly how resolve the problem

**Comment by alexsanford** [comment]
@xxhenryxx The problem is with the WPML plugin, not WP Job Manager. Please get in touch with them for further support.

https://wpml.org/forums/forum/english-support/
